JavaScript/JavaScriptSDK/appInsights.ts

```diff
@@ -190,16 +190,18 @@ module Microsoft.ApplicationInsights {
             if (!Telemetry.PageViewPerformance.isPerformanceTimingSupported()) {
                 // no navigation timing (IE 8, iOS Safari 8.4, Opera Mini 8 - see http://caniuse.com/#feat=nav-timing)
                 _InternalLogging.throwInternalNonUserActionable(LoggingSeverity.CRITICAL,
```

Comment:
Do we still need to report it to the customer? It is a known issue that neither customer nor we can address. So perhaps we can omit it. Also zero will always mean "navigation API not supported" so you can query this information without having this error message